packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx

```diff
@@ -256,7 +261,9 @@ const ConditionsWidget: React.FC<ConditionsWidgetProps> = ({
                   </ResponsiveWrapper>
                 )}
               />
-              {errors?.conditionName && <p className={styles.errorMessage}>{errors?.conditionName?.message}</p>}
+              {errors?.conditionName && (
+                <p className={styles.errorMessage}>{errors?.conditionName?.message as string}</p>
```

Comment:
I imagine the `as string` type casting is necessitated by not using the most ideal type annotations.